packages/schemer/src/index.ts

```diff
@@ -165,7 +166,7 @@ export default class Schemer {
         const fieldPath = schemaPointerToFieldPath(jsonPointer);
         assets.push({
           fieldPath,
-          data: get(data, fieldPath),
+          data: get(data, lowerFirst(fieldPath)),
```

Comment:
Is this always true?

Reply:
In this use case: yes; in all of them: definitely can't guarantee it. What hoping to see how AJV handles, but there's a lot of layers there.

Feels not great, but absent of reimplementing `get`, this should handle both:
```get(data, lowerFirst(fieldPath)) || get(data, fieldPath)```